systemlink/clients/dataframe/_data_frame_client.py

```diff
@@ -1,5 +1,16 @@
+# TODO: Wrap Uplink decorators to add typing information
```

Comment:
FYI: It's also possible to create a pyi stub like [this one](https://github.com/ni/nisystemlink-clients-python/blob/master/systemlink/stubs/events.pyi) that adds typing information to a library without modifying that library. It may not be worth it though... But there's [a command you can run](https://mypy.readthedocs.io/en/stable/stubs.html) to generate the initial stub file for you to fill in.

Reply:
There's a good chance I'll end up wrapping their decorators anyway to simplify things for our clients. But the stubs could be helpful. I'll look into it.